### DISCUSSION TITLE
What do the other players see?

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#831] [(A) Anything]
I was trolling in prodigy as a level 12 then just eliminating level 100's by doing 87978 damage...

but then I thought "what doe the opponent see"

so can anyone tell me or show me what they see?

**Comment by ghost** [comment]
Do they see me doing 854793878465 damage?

**Reply by PatheticMustan** [reply]
no

**Reply by ghost** [reply]
What do they see then?

**Reply by ghost** [reply]
@PatheticMustan What do they see me do?

**Reply by PatheticMustan** [reply]
idk, probably the normal amount

level 100 hacks still apply normally, just not modifiers or god mode
